misaligned.cpp

```diff
@@ -16,6 +20,7 @@ int printColorMap() {
 void testPrintColorMap() {
     std::cout << "\nPrint color map test\n"; 
     int result = printColorMap();
+    assert(colorCount(0,0) == 0);

```

Comment:
Let the assert always convey the "true" requirement. In this case, the first pair number needs to be `1` and not `0`.
...and correct the `colorCount` with `+ 1` to return the same